config/internal/mariadb/mariadb-sa.yaml.tmpl

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mariadb-sa
```

Comment:
can you follow the -{{ .Name }} pattern like all the other resources, same for minio sa

Reply:
Done, updated to have the  -{{ .Name }} pattern.